lib/process-services-cloud/src/lib/form/components/widgets/date/date-cloud.widget.html

```diff
@@ -12,6 +12,7 @@
                 [id]="field.id"
                 [formControl]="dateInputControl"
                 [placeholder]="field.placeholder"
+                [value]="field.value"
```

Comment:
needs unit tests to replicate the problem and fix

Reply:
the `[formControl]="dateInputControl"` already binds the value, so I believe this change is not required